AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_ExposureControlAdded.py

```diff
@@ -149,35 +239,285 @@ def AtomEditorComponents_ExposureControl_AddedToEntity():
         # 7. Verify Exposure Control component is enabled.
         Report.result(Tests.exposure_control_enabled, exposure_control_component.is_enabled())
 
-        # 8. Enter/Exit game mode.
+        # 8. Toggle the Enable parameter.
+        # Set the Enable parameter to False.
+        exposure_control_component.set_component_property_value(
+            AtomComponentProperties.exposure_control('Enable'), False)
+        Report.result(
+            Tests.toggle_enable_parameter_off,
+            exposure_control_component.get_component_property_value(
+                AtomComponentProperties.exposure_control('Enable')) is False)
+        # Set the Enable parameter to True.
```

Comment:
nit: blank line before substep comments separates the code for readability

Reply:
Blank lines added.